### DISCUSSION TITLE
Bosean FS-600: Can't flash

### DISCUSSION THREAD

**Discussion by catgirl2137** [Gissio/radpro#104] [Q&A] [answered]
Hello
I recently tried to flash RadPro 2.0 to a Bosean FS-600 geiger counter.
Even though I followed all the guidelines, the process errored out while creating a bakup.
Because this is my first time dealing with STM32 based devices, I have no idea what could be the reason for such error, and I would very much appreciate any help. 
Down below is a screenshot of the error I'm getting
![Error_FS600](https://github.com/Gissio/radpro/assets/163208760/5e276924-5885-4638-96b7-1251461a807b)
and a photo of the circuit board. 
![20240618_185313](https://github.com/Gissio/radpro/assets/163208760/879ea21a-91b7-40db-9ba7-974b7d2cdca4)
![20240618_190057](https://github.com/Gissio/radpro/assets/163208760/55872720-877f-4b86-a9ad-3c5153269f13)
It differs from the one in the example by capacitor/resistor placement, but the IC seems to be the same type.
Thank You in advance!

**Comment by Gissio** [comment]
First, check the electrical connections with a multimeter.

If it still fails, try keeping the ok/power button pressed while flashing, with the main battery connected.

**Reply by Gissio** [reply]
Please let us know if that worked, and exactly how.

The installation instructions might need to be corrected.

**Reply by catgirl2137** [reply]
Thank you for your fast response!
Already checked the connections - all ok
ST-link V2 driver is installed
Tried flashing multiple times while holding power button, both with and without battery - same result

**Reply by Gissio** [reply]
Unfortunately I can't help you any further.

**Reply by catgirl2137** [reply]
Is there any chance of an IC pin that has to be pulled up or down to enable programming?

**Reply by Gissio** [reply, marked as answer]
I don't own a Bosean FS-600. Perhaps someone with such a device in this forum can help you any further.

**Reply by DG0MG** [reply]
I have 3 Models of FS-600 and i can confirm there are no bigger problems to flash the device.
Sometimes you have to try a second time or unplug the USB programmer and plug it back in.

Show a photo of how you connected the programmer to the 4 pins of the programming header. Sure, all is correct?
Does the FS-600 start when you connect the programmer?

For the FS-600 look here too: [https://www.geigerzaehlerforum.de/index.php/topic,1068.0.html](https://www.geigerzaehlerforum.de/index.php/topic,1068.0.html)

**Reply by J0s3f** [reply]
For the FS-5000, the device boots while connected to the STLINK and then cannot be flashed anymore. I can flash it reliably without issues if I start the flashing process directly after connecting the STLINK to USB. If I wait too long, it won't work anymore. 
Maybe it's a similar issue on the FS-600?
